src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -36,4 +39,17 @@ public void execute_listIsFiltered_showsEverything() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
         assertCommandSuccess(new ListCommand(), model, ListCommand.MESSAGE_SUCCESS, expectedModel);
     }
+
+    @Test
+    public void execute_listIsFiltered_showsOrderedList() {
+        assertCommandSuccess(new ListCommand(), model, ListCommand.MESSAGE_SUCCESS, expectedModel);
+
+        ObservableList<Person> checkSorted = model.getFilteredPersonList();
+
+        Person temp = checkSorted.get(0);
```

Comment:
Shorter alternative can be to create a PersonList, sort it using a `Comparator`, then just test using `.equals`